apps/backend/agents/safety/pii_detector.py

```
@@ -0,0 +1,258 @@
+        ),
+        "credit_card": re.compile(
+            r'\b\d{4}[-\s]?\d{4}[-\s]?\d{4}[-\s]?\d{4}\b'
+        ),
+        "address": re.compile(
+            r'\b\d{1,5}\s+[A-Za-z]+\s+(?:Street|St|Avenue|Ave|Road|Rd|Boulevard|Blvd|Lane|Ln|Drive|Dr|Court|Ct|Circle|Cir)\b',
+            re.IGNORECASE
+        ),
+        "zipcode": re.compile(
+            r'\b\d{5}(?:-\d{4})?\b'
+        ),
+    }
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)